src/nvim/options.lua

```diff
@@ -834,7 +836,7 @@ return {
       abbreviation = 'bh',
       cb = 'did_set_bufhidden',
       defaults = { if_true = '' },
-      values = { 'hide', 'unload', 'delete', 'wipe' },
+      values = { '', 'hide', 'unload', 'delete', 'wipe' },
```

Comment:
This will probably add an empty string to `opt_bh_values`, which is probably not intended. Maybe add an empty string check in `gen_options.lua`?

Reply:
Yes